nodejs/week1/src/backend/app.js

```diff
@@ -0,0 +1,83 @@
+const express = require("express");
+const app = express();
+
+// import data here
+const meals = require("./data/meals.json");
+const reservations = require("./data/reservations.json");
+const reviews = require("./data/reviews.json");
+const warmup = require("./data/warmup.js");
+
+//-----the result of the of the circle measurements in the server----------------
+
+console.log(warmup); 
+
+//----------Meals sharing Web ---------------------------
+
+app.get("/", async (request, response) => {
+    response.send("Meals Sharing Web App");
```

Comment:
here is a small typo, that leads to a failed test! i know it has nothing to do with your code, but try to run the test suite one more time, to see what i mean ;)